### ISSUE TITLE
Add Talk description section for Talks

### ISSUE THREAD

**Issue by ananyo2012** [pythonindia/inpycon2018#77]
We want to add a modal/pop-up containing the description of the talk and the speaker description. When someone clicks over a talk in the schedule he/she should be able to see the pop-up. The section for the conference schedule is presently commented out https://github.com/pythonindia/inpycon2018/blob/master/index.html#L202-L669

Presently the schedule is all hard-coded. To make it simpler, you need to parse the data from a json file and loop over the data to display the schedule and the talk descriptions

You may want to show a mockup before starting the work

**Comment by dbads** [comment]
yeah.. that would be awesome :+1:

**Comment by thisisayush** [comment]
Last year, I worked on creating the scraper for fetching the talk details from the proposals page and storing 
 it in JSON, instead of manually creating a JSON File. Once we have the JSON file, we can read it from there and show the popup. But will it not decrease the SEO?

**Comment by ananyo2012** [comment]
@thisisayush Can we reuse the scraper? That would really save time for collecting the data. I guess the SEO won't must be affected since the main part of SEO are the schema markups and the meta data. We will make a json-ld doc with the markup. Will create a separate issue for that.

**Comment by thisisayush** [comment]
@ananyo2012 I can re-write the scraper for the purpose, that's not an issue.
I didn't know about JSON-LD, after a quick google it seems the key I have been missing with using JSON for data in static websites while considering SEO.

**Comment by tulikavijay** [comment]
Is this open? I can take it up

**Comment by ananyo2012** [comment]
@thisisayush Are you free to work on this? The first thing you need to do is to remove all the hard coded schedule code and move the data to a json file and display the schedule by parsing data from json using javascript.

**Comment by Niveshkrishna** [comment]
@ananyo2012 is someone working on this?

**Comment by realslimshanky** [comment]
@Niveshkrishna there's no active person assigned to this issue. Please take it up.

**Comment by ananyo2012** [comment]
I think we need to re think on this. The usual way is to just link the proposal from CFP page. Is this a good idea? Tagging @pythonindia/pycon-2018-team for suggestion

**Comment by vipulgupta2048** [comment]
The usual way can be used for backup if no one takes it up. 
I think parsing data from JSON would save us a lot of work and further maintenance.  
I really like how DevConf India did their schedule page. Check it out https://devconfin2018.sched.com/

**Comment by ananyo2012** [comment]
They are showing it in a separate page. The Website is a PWA. I think we
should use a modal instead of adding the link to CFP. This will keep the
user within the website instead of moving to another page. What do you
think?

**Comment by akshat0047** [comment]
I'll be working on this issue, first will be considering the usual json method and once it work fine, will go for the scraping part.
@ananyo2012 @vipulgupta2048

**Comment by vipulgupta2048** [comment]
@akshat0047  Thanks for taking it up. 
Can you describe the usual JSON method that you will be following? 
It's just that you would have to populate the JSON one way or the other.

**Comment by akshat0047** [comment]
Usual method, i.e. manually edit the json whenever adding a new schedule for the basic structure.
@vipulgupta2048

**Comment by ananyo2012** [comment]
This isn't needed anymore. Not feasible to get all speaker images at this point. Hence closing this.
